packages/at_secondary_server/lib/src/verb/handler/monitor_verb_handler.dart

```diff
@@ -13,7 +17,7 @@ class MonitorVerbHandler extends AbstractVerbHandler {
 
   late InboundConnection atConnection;
 
-  String? regex;
+  late String regex;
```

Comment:
neither the inbound connection nor the regex should be instance variables

Reply:
As discussed, the changes are deferred